konfera/templates/konfera/event_speakers.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'konfera/base.html' %}
+
+{% block title %}<h1>{{ event }} Speakers</h1>{% endblock %}
```

Comment:
Dont forget that the title block is in `<head>` tag. Use `<title>` instead of `<h1>`.